components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1504,6 +1504,7 @@ where
         ctx: &mut ApplyContext<EK, W>,
         req: &Request,
     ) -> Result<()> {
+        PEER_WRITE_CMD_COUNTER.put.inc();
```

Comment:
How about using local metrics?

Reply:
`auto_flush_from!` already make it be local metrics.